src/lib/viewers/media/Settings.js

```diff
@@ -292,11 +292,15 @@ class Settings extends EventEmitter {
      * @return {void}
      */
     setMenuContainerDimensions(menu) {
-        // NOTE: need to explicitly set the dimensions in order to get css transitions. width=auto doesn't work with css transitions
-        this.settingsEl.style.width = `${menu.offsetWidth + 18}px`;
+        const { children } = menu;
+
+        // If we have enough children to require scrolling, take into account scroll bar width.
+        const scrollPadding = children.length > 7 ? 32 : 18;
```

Comment:
Extract these magic numbers to constants

Reply:
Would looking at scrollHeight help? 

`The Element.scrollWidth read-only property is a measurement of the width of an element's content, including content not visible on the screen due to overflow.`

https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollWidth